sunpy/timeseries/timeseries_factory.py

```diff
@@ -338,6 +341,46 @@ def _parse_args(self, *args, **kwargs):
                 raise type(e)(msg) from e
         return all_ts
 
+    def _parse_ts_results(self, r, **kwargs):
```

Comment:
Is this just a code refactor because it was messy? I don't see any changes regarding uris

Reply:
Yes, just refactoring. It use to be code under `_parse_path(self, path, **kwargs)`. I'm pulling it out because I use the same code for `_parse_uri(self, obj_list, **kwargs)`.